public/js/discover/searchList.js

```diff
@@ -83,15 +83,10 @@ export function selectCard(id = null, scroll = false) {
  * @return {string} The formatted phone number.
  */
 function formatPhoneNumber(number) {
-  if (number.length == 9) {
-    const areaCode = number.substring(0, 3);
-    const exchangeCode = number.substring(3, 6);
-    const lineNumber = number.substring(6, 9);
-    return `(${areaCode})-${exchangeCode}-${lineNumber}`;
-  } else if (number.length == 10) {
-    const countryCode = number.substring(0, 1);
-    return `+${countryCode} ${formatPhoneNumber(number.substring(1, 10))}`;
-  } else {
-    return number;
+  const regexMatch = number.match(/^(1|)?(\d{3})(\d{3})(\d{4})$/);
+  if (regexMatch) {
+    const countryCode = regexMatch[1] ? '+1 ' : '';
```

Comment:
So are we converting all existing country codes to +1 here?

If we're doing that, why not drop the country code and we should validate phone number from the edit view.